compiler/rustc_passes/src/dead.rs

```diff
@@ -242,6 +242,24 @@ impl<'tcx> MarkSymbolVisitor<'tcx> {
         }
     }
 
+    fn should_ignore_impl(&self, impl_def_id: DefId) -> bool {
```

Comment:
Can we add a doc comment explaining the purpose of this function and why those particular traits were chosen? (Perhaps linking to this PR)

Reply:
Fixed, along with your other suggestion.